### ISSUE TITLE
Project unmaintained?

### ISSUE THREAD

**Issue by brianjcohen** [xndcn/symbols-tree-view#171]
It seems the original author of this project may have moved on; the forks and pull requests are starting to pile up.  Maybe @xndcn would be interested in turning the project over to a new maintainer(s) ?

I'm not much of a CoffeeScript expert but I'd be willing to help out. I rely heavily on this plugin and I'd hate to see it stop getting improvements.

**Comment by Joouis** [comment]
@brianjcohen Hey buddy, I'm interested in this project too, and already port it to my personal stuff. If you would like to code in ES6, maybe we can maintain it together ; )

**Comment by brianjcohen** [comment]
@Joouis it might make more sense to hitch our wagons to this fork: https://github.com/nicolashainaux/symbols-tree-nav

**Comment by nicolashainaux** [comment]
@brianjcohen @Joouis Frankly I'd prefer to see symbols-tree-view alive again, merging new PR, enhancements, bug fixes...

I did publish a forked package because not much seems to happen since months now, and because such a reliable and at least up to date "class-navigator" feature is vital for my use of atom. Maybe the new features and bug corrections in symbols-tree-nav can be merged back to symbols-tree-view at some point in the future? In the meanwhile, PR, bug corrections and enhancements are welcome on symbols-tree-nav. I am not coffee script expert neither, and if anyone having better skills with it want to maintain it, it's open too.

By the way, there's an issue I had to reopen and cannot correct (actually cannot reproduce the bug), so any valuable information on it is welcome too (more details about if/how the bug can be reproduced etc.).

**Comment by onetrev** [comment]
There is also [Symbols List](https://github.com/7ute/symbols-list), which might be an option too for now. A bit different than Symbols Tree View, but same general idea. Might work for some folks for now. Almost wish this overall functionality was built into Atom, it's such a key coding tool.

**Comment by Joinyy** [comment]
@onetrev Yes I also wished it was part of atom core packages. Also because you can already browse symbols with Ctrl-R it should not be that big deal to display them in a nice dock.

**Comment by Joouis** [comment]
@brianjcohen @nicolashainaux @onetrev @Joinyy 
Hey buddies, check [structure-view](https://atom.io/packages/structure-view) out, hope you can enjoy it!

**Comment by nicolashainaux** [comment]
Yes, it looks good! Moreover it's embedded in a dock at right, so it's resizeable.
The javascript files look better than in symbols-tree-nav/view, but the support for python is not enough.
I hope it'll support more languages soon.

**Comment by Joouis** [comment]
@nicolashainaux Of course I do have plan to support more and more popular languages like Python when I design the architecture of this plugin.
Javascript files look better because JS is my working choice now, so I started from JS, HTML and CSS.
More languages support need more efforts, I'm thinking about how to draw peoples' attention and get contributors in now.

**Comment by onetrev** [comment]
Hey @Joouis, this is very exciting! Thanks for the information. I had been looking for alternatives again just last week and unfortunately didn't notice your project.

As to how to get more peoples' attention, that is a tough one. I'm not sure how you get the folks at Atom to add something to the "Featured Packages" but in my mind yours should be a candidate for it. This is key and popular developer functionality, since Symbols Tree View has over 100K downloads, but since that package is only minimally updated now, folks need to know about the alternatives out there!

**Comment by brayfe** [comment]
Looks like there is still no maintainer for this project, 2 years later.  I think this thread should now live as a place to list alternatives to this plugin.  +1 for the idea that Atom includes this functionality as a core package.
